exercises/hamming/lib/example.dart

```diff
@@ -1,7 +1,9 @@
 class Hamming {
   int distance(String a, String b) {
-    if (a.length != b.length) {
-      throw new ArgumentError("DNA strands must be of equal length.");
+    if ((a.length == 0 && b.length > 0) || (a.length > 0 && b.length == 0)) {
```

Comment:
Suggestion: Prefer `String.isEmpty` instead of `String.length==0`, Same for `String.isNotEmpty`
```suggestion
    if ((a.isEmpty && b.isNotEmpty) || (a.isNotEmpty && b.isEmpty)) {
```

Reply:
That's what I initially used, but I recall getting an error. So I changed it to this usage. Reverting to those methods doesn't break anything, maybe there was some other change that caused the error?

Anyway, suggestion implemented